DTS-STN/secure-client-hub
# Update dependencies

### Changelog - "fix:" for bug fixes, "feat:" for features. Read more about Conventional Commits at https://www.conventionalcommits.org/en/v1.0.0/#summary

### Description of proposed changes:
This PR addresses some upgrades to our dependencies as pointed out by Snyk. There were some significant changes to our package-lock that would make fixing merge conflicts a pain so this PR serves to update all those dependencies in one go. I've also updated Cypress in this PR

### What to test for/How to test
1. Pull in branch
2. Type `npm i`
3. Type `npm run dev`
4. Ensure app runs as expected
